website/docs/docs/build/metricflow-commands.md

```diff
@@ -78,10 +78,9 @@ You can use the `dbt sl` prefix before the command name to execute them in the d
 - [`query`](#query) &mdash; Query metrics, saved queries, and dimensions you want to see in the command line interface. Refer to [query examples](#query-examples) to help you get started.
 - [`export`](#export) &mdash;  Runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.
 - [`export-all`](#export-all) &mdash; Runs exports for multiple saved queries at once, saving time and effort.
-
+- [`validate-configs`](#validate-configs) &mdash; Validates semantic model configurations.
```

Comment:
it should be just `validate` not `validate-configs`

Reply:
fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/5930/commits/51a821ce13b69866bcf690b9f3761312725fef27